consul/consul.go

```diff
@@ -13,9 +12,9 @@ import (
 
 type ConsulServices interface {
 	GetAllServices() ([]*consulapi.CatalogService, error)
-	GetServices(name tasks.AppId) ([]*consulapi.CatalogService, error)
-	Register(task *tasks.Task, app *apps.App) error
-	Deregister(serviceId tasks.Id, agentAddress string) error
+	GetServices(name string) ([]*consulapi.CatalogService, error)
+	Register(task *apps.Task, app *apps.App) error
```

Comment:
this will end up with a big code refactor, I would suggest doing this in the future. :)